frontend/src/employee-mobile-frontend/components/UnitList.tsx

```diff
@@ -0,0 +1,66 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import { ContentArea } from 'lib-components/layout/Container'
+import { FixedSpaceColumn } from 'lib-components/layout/flex-helpers'
+import { H1, H2 } from 'lib-components/typography'
+import { Gap } from 'lib-components/white-space'
+import React, { useContext, useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components'
+import { useTranslation } from '../state/i18n'
+import { UnitContext } from '../state/unit'
+import { renderResult } from './async-rendering'
+import { TopBar } from './common/TopBar'
+
+export default React.memo(function UnitList() {
+  const { i18n } = useTranslation()
+  const { units, reloadUnits } = useContext(UnitContext)
+  useEffect(reloadUnits, [reloadUnits])
+
+  return (
+    <>
+      <TopBar title={i18n.units.title} />
+      <Gap size="xs" />
+      {renderResult(units, (units) => (
+        <ContentArea opaque>
+          <CenteredH1 noMargin>{i18n.units.title}</CenteredH1>
+          <Gap size="L" />
+          <FixedSpaceColumn spacing="m">
+            {units.map(({ id, name, groups }) => {
+              const [present, total] = groups.reduce(
+                ([present, total], group) => [
+                  present + group.presentChildren,
+                  total + group.totalChildren
+                ],
+                [0, 0]
+              )
+
+              return (
+                <UnitContainer key={id} to={`/units/${id}`}>
+                  <H2 noMargin>{name}</H2>
+                  <Gap size="xxs" />
+                  <FixedSpaceColumn spacing="xxs">
+                    <span>{`${i18n.units.children} ${present}/${total}`}</span>
+                  </FixedSpaceColumn>
```

Comment:
I think this could be simplified by adding `gap: ${defaultMargins.xxs};` to `UnitContainer` and removing the `FixedSpaceColumn` here altogether since it does not seem to add any benefit..?